editions/tw5.com/tiddlers/nodejs/TiddlyWiki on Node.js.tid

```diff
@@ -1,15 +1,20 @@
 created: 20131129094353704
-modified: 20200318115527226
+modified: 20220612092156888
 tags: Platforms
 title: TiddlyWiki on Node.js
 type: text/vnd.tiddlywiki
 
+\rules except wikilink
```

Comment:
Why are you adding this rule? It's not consistent with other documentation tiddlers.

Reply:
I think topics like this need a bit more structure, to be understandable for new users. Especially TiddlyWiki and other CamelCase links adds a lot of noise. I tried to reduce those links to 1 appearance near the end of the tiddler.